src/services/schema/update/analyzer/sequelize-column-type-getter.js

```diff
@@ -122,8 +122,6 @@ function ColumnTypeGetter(databaseConnection, schema, allowWarning = true) {
       case 'DOUBLE':
       case 'DOUBLE PRECISION':
       case typeContains(type, 'DECIMAL'):
-      case 'MONEY': // MSSQL type
```

Comment:
It would be super great to have non-regression test on this. We are changing the behaviour there, and no tests are broken, meaning this section is not tested correctly.